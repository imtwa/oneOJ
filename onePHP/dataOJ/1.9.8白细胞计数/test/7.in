111
20.7874 12.7483 65.8728 24.7181 77.8034 57.6097 53.6541 64.8583 20.0221 73.8141 24.0163 63.7064 61.4263 93.667 66.4225 29.293 44.1889 63.043 96.8603 47.4743 11.2963 18.6273 58.1607 92.3234 60.0931 62.5313 57.3876 76.4238 67.8125 11.0633 37.6527 46.7737 26.6964 95.7405 24.0663 71.0277 68.3172 77.6985 64.7594 54.7862 76.5443 84.7317 77.4834 54.204 26.0972 26.7274 44.5761 95.5541 28.7579 76.3932 47.0966 19.0768 64.5482 78.375 40.8332 32.7821 68.3044 19.0372 50.5765 49.7815 70.2305 51.1322 46.2529 59.4802 45.9318 41.4958 43.0682 90.2986 36.425 42.4021 89.2609 60.2868 21.0238 79.5419 82.3994 46.0807 88.2255 65.0934 64.357 67.8715 27.4653 21.1218 77.4715 69.3779 39.2596 52.7946 92.423 50.4365 26.7721 62.1046 94.3367 70.5484 90.8737 44.4287 57.4095 52.4455 34.834 34.1271 16.1918 36.9861 63.4919 85.6601 32.9498 57.3569 79.6042 97.7341 80.3062 72.5604 83.4367 59.1134 33.028
