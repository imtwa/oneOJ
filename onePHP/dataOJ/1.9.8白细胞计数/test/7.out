55.83 44.53